.gitignore

```diff
@@ -10,6 +10,9 @@ tmtags
 \#*
 .\#*
 
+## RUBYMINE
```

Comment:
This is RUBYMINE-specific what about all other development-environments? Can you build a solution on your own or is it needed to add it to `aruba`?